docs/cloud-configuration/configuration-aws.md

```diff
@@ -1,45 +1,15 @@
-> ⚠️ Note from the developers: We are currently in the process of moving our documentation and so the below documentation is only partially correct. For more information on this tool please checkout our [plugin documentation](https://github.com/bitops-plugins/aws).
+> ⚠️ Note from the developers: We are currently in the process of moving our documentation and so the below documentation is only partially correct. For more information on this tool please check out our [plugin documentation](https://github.com/bitops-plugins/aws).
 
 # AWS
 
-> ⚠️ `bitops.config.yaml` is not yet supported for AWS ([TODO](https://github.com/bitovi/bitops/issues/15)). All configuration must be done with environment variables.
+> ⚠️ `bitops.config.yaml` is not yet supported for AWS ([TODO](https://github.com/bitovi/bitops/issues/15)). All configurations must be done with environment variables.
 
 ## Configuration
 
--------------------
-### aws_access_key_id
-* **BitOps Property:** [TODO](https://github.com/bitovi/bitops/issues/15)
-* **Environment Variable:** `AWS_ACCESS_KEY_ID`
-* **default:** `""`
-* **required:** yes
+| Item                  | BitOps Property                                    | Environmental Variable | Description                                                  | Default | Required |
+| --------------------- | -------------------------------------------------- | ---------------------- | ------------------------------------------------------------ | ------- | -------- |
+| aws_access_key_id     | [TODO](https://github.com/bitovi/bitops/issues/15) | AWS_ACCESS_KEY_ID      | Specifies an AWS access key associated with an IAM user or role. See [AWS official documentation](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys) | `null`  | Yes      |
```

Comment:
@mickmcgrath13 @PhillypHenning Do we know what these `TODO` links really mean?
Any history context and can we remove those?

There is no `BitOps Property` columns in the other table.

Reply:
Those TODO's link to issue #15 and were in the original documentation

Reply:
So this one has an 'Extra' column, named Item.  The property column is also called 'BitOps Property' in this table.   As I formatted the information into a table, I wanted to ensure that I kept the same naming conventions as there were original.